src/main/java/org/sagebionetworks/bridge/models/schedules/ABTestScheduleStrategy.java

```diff
@@ -31,10 +32,8 @@ public Schedule getScheduleForUser(SchedulePlan plan, ScheduleContext context) {
         }
         // Randomly assign to a group, weighted based on the percentage representation of the group.
         ABTestGroup group = null;
-        
-        // A random(ish) number that is stable for a given user against a given schedule plan.
-        long seed = Math.abs(plan.getGuid().hashCode() + 
-                context.getCriteriaContext().getHealthCode().hashCode());
+        long seed = UUID.fromString(plan.getGuid()).getLeastSignificantBits()
+                + UUID.fromString(context.getCriteriaContext().getHealthCode()).getLeastSignificantBits();        
```

Comment:
Looks like you accidentally reverted your Guid changes here.

Reply:
restored